package.json

```diff
@@ -1,16 +1,21 @@
 {
-    "name": "zetkin-organizer-dashboard",
-    "author": "Richard Olsson",
-    "repository": {
-        "type": "git",
-        "url": "git://github.com/zetkin/organizer-dashboard.git"
-    },
-    "version": "0.0.0",
-    "main": "app",
-    "scripts": {
-        "start": "node ."
-    },
-    "dependencies": {
-      "express": "~4.12.0"
-    }
+  "name": "zetkin-organizer-dashboard",
+  "author": "Richard Olsson",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/zetkin/organizer-dashboard.git"
+  },
+  "version": "0.0.0",
+  "main": "app",
+  "scripts": {
+    "start": "node bin/server.js",
+    "test": "tape tests/test.js"
```

Comment:
protip: you don't need tape globally installed for this to work - when running `npm test` it's adding `node_modules/.bin` to ENV so it's available.